cartridges/int_stickyio_sfra/cartridge/templates/resources/stickyio.properties

```diff
@@ -46,6 +46,7 @@ link.manage=Manage
 button.subscriptionmanagement.confirm=CONFIRM
 button.subscriptionmanagement.cancel=CANCEL
 button.subscriptionmanagement.close=CLOSE
+button.subscriptionmanagement.update=Update
```

Comment:
The other items are in uppercase, so, I think we should do uppercase here too to be consistent: button.subscriptionmanagement.update=UPDATE

Reply:
I would not recommend that as this button in used on the edit screen.  The other (base) edit address screens use the same case.  So if we change this it will not be consistent with the other (base functionality) buttons.  The  uppercase buttons you mentioned are used on a popup and i do not know why uppercase was used.

Reply:
Agreed, if we set it to uppercase it would be a screaming button lol